packages/ag-grid-community/src/dragAndDrop/rowDragComp.ts

```diff
@@ -206,6 +185,27 @@ export class RowDragComp extends Component {
         this.beans.dragAndDrop!.addDragSource(this.dragSource, true);
     }
 
+    private getDragItemName(draggingEvent: RowDraggingEvent | null | undefined, translate: LocaleTextFunc): string {
+        const dragItem = draggingEvent?.dragItem || this.getDragItem();
+        const dragItemCount = (draggingEvent?.dropTarget?.rows.length ?? dragItem.rowNodes?.length) || 1;
```

Comment:
```suggestion
        const dragItemCount = (draggingEvent?.dropTarget?.rows.length ?? dragItem.rowNodes?.length) ?? 1;
```
do we need to do a minimum 1 item count?

Reply:
Yeah I would not change this - as we call a user callback like this and we do not want to add a breaking change